simple_network_sim/loaders.py

```diff
@@ -111,8 +111,11 @@ def genGraphFromContactFile(commutes: pd.DataFrame) -> nx.DiGraph:
     """
     G = nx.convert_matrix.from_pandas_edgelist(commutes, edge_attr=True, create_using=nx.DiGraph)
     for edge in G.edges.data():
-        assert edge[2]["weight"] >= 0.0
-        assert edge[2]["delta_adjustment"] >= 0.0
+        if "weight" not in edge[2] or edge[2]["weight"] < 0.0:
+            raise ValueError("missing weight or weight less than zero")
+        if "delta_adjustment" not in edge[2] or edge[2]["delta_adjustment"] < 0.0:
+            raise ValueError("missing delta_adjustment or delta_adjustment less than zero")
+
```

Comment:
I thought something like this as well, but was being bashful - a default to 1.0 would be preferable (but I didn't want to demand things in this PR)

Reply:
Previously, we would raise a KeyError because edge[2]["weight"] was used without  checking if the key existed or not. So I just made the behaviour more explicit.

I can default them to 1.0 when we load them, it's not a problem at all.

Reply:
Actually, let me do a separate PR for that